src/common/components/UnexpectedError.js

```diff
@@ -32,12 +32,7 @@ const UnexpectedError = () => {
       justifyContent="center"
       sx={{ height: '80vh' }}
     >
-      <Stack
-        component={Paper}
-        elevation={0}
-        sx={theme => ({ maxWidth: 'md', padding: 3 })}
-        alignItems="center"
-      >
+      <Stack component={Paper} elevation={0} alignItems="center">
```

Comment:
issue: you removed the spacing entire. Please apply Jose's suggestion.

Reply:
Looks like something got mangled during the merge/change. The only lime that should change is
```
        sx={theme => ({ maxWidth: 'md', padding: theme.spacing(3) })}
```

Reply:
sorry about that, my revert was a bit buggy; should be fixed now